versions/version3/version3.js

```diff
@@ -1,44 +1,71 @@
+const DEFAULT_OPTIONS = {
+  initialColor: "#e4e6e3",
+  secondColor: "#6cfe6b",
+  settledColor: "#00dd00",
+  bold: true,
+  shading: true,
+  fontSize: 20,
+  minLength: 5,
+  maxLength: 20,
+  slowestInterval: 500,
+  fastestInterval: 50,
+  columnWidthTweak: 0.85,
+  totalStreamMultiplier: 1,
+};
+
+const CHARACTERS =
+  "MATRIXMATRIXMATRIXMATRIXMAØ1Ø1Ø1Ø1Ø#$%@&#$%@&ｦｧｨｩｪｫｬｭｮｯｱｲｳｴｵｶｷｸｹｺｻｼｽｾｿﾀﾁﾂﾃﾄﾅﾇﾈﾉﾊﾋﾌﾍﾎﾏﾐﾑﾒﾓﾔﾕﾖﾗﾘﾙﾚﾛﾜﾝｦｧｨｩｪｫｬｭｮｯｱｲｳｴｵｶｷｸｹｺｻｼｽｾｿﾀﾁﾂﾃﾄﾅﾇﾈﾉﾊﾋﾌﾍﾎﾏﾐﾑﾒﾓﾔﾕﾖﾗﾘﾙﾚﾛﾜﾝｦｧｨｩｪｫｬｭｮｯｱｲｳｴｵｶｷｸｹｺｻｼｽｾｿﾀﾁﾂﾃﾄﾅﾇﾈﾉﾊﾋﾌﾍﾎﾏﾐﾑﾒﾓﾔﾕﾖﾗﾘﾙﾚﾛﾜﾝｦｧｨｩｪｫｬｭｮｯｱｲｳｴｵｶｷｸｹｺｻｼｽｾｿﾀﾁﾂﾃﾄﾅﾇﾈﾉﾊﾋﾌﾍﾎﾏﾐﾑﾒﾓﾔﾕﾖﾗﾘﾙﾚﾛﾜﾝ";
+
+const getRandomChar = () => {
+  const randNum = Math.floor(Math.random() * CHARACTERS.length);
+  return CHARACTERS.charAt(randNum);
+};
```

Comment:
Refactor 2: Extract `characters` and `getRandomChar` outside of the `DigitalRain` class.